templates/user/notification/notification_div.tmpl

```diff
@@ -14,22 +14,22 @@
 				<form action="{{AppSubUrl}}/notifications/purge" method="POST" style="margin-left: auto;">
```

Comment:
`style="margin-left: auto;"` should this be fixed together? since it's a refactoring.

Reply:
Might as well but it appears in more places, not only here.

Reply:
Fixed in https://github.com/go-gitea/gitea/pull/24639/commits/f5c46f165c4c8bd50b0d0944663d6aabd29bf057 and also fixed 5 other instances in the templates.